src/PageRouter.js

```diff
@@ -0,0 +1,36 @@
+import { React } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import HomePage from "./components/pages/HomePage";
+import WelcomePage from "./components/pages/WelcomePage";
+import CreateInvestorPage from "./components/pages/CreateInvestorPage";
+import BacktestingPage from "./components/pages/BacktestingPage";
+import TransactionHistoryPage from "./components/pages/TransactionHistoryPage";
+import AccountPage from "./components/pages/AccountPage";
+import SignInPage from "./components/authentication/SignInPage";
+
+export function PageRouter() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<WelcomePage />} />
```

Comment:
Should the `WelcomePage` route be in both `PageRouter` and `UnauthenticatedPageRouter`. I would think it should only be on the `UnauthenticatedPageRouter`

Reply:
Upon signing out, the user is redirected to the WelcomePage, regardless if they are authenticated or not. If WelcomePage is only in UnauthenticatedPageRouter, the screen goes blank upon sign out. Currently trying to find a workaround for this.